### PR TITLE
Add links to Discord and Matrix

### PR DESCRIPTION
**Pull request by i2**
Also add a cool stargazer chart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,6 +260,9 @@ gdb foo.com -ex 'add-symbol-file foo.com.dbg 0x401000'
     for a community supported version of Cosmopolitan that works on Windows
     Vista and Windows 7.
 
+## Community
+Please feel free to join us on [Matrix](https://matrix.to/#/#redbean:matrix.org) or [Discord](https://discord.gg/sN53mQMt) for further discussion and support!
```
Comment (jart) [inline README.md:264]: Please put these in a bulleted list. The Discord needs to come first.

File: README.md [hunk 2]
```diff
@@ -273,3 +276,9 @@ the following groups and individuals:
 - [Wasmer](https://wasmer.io/)
 
 For publicly sponsoring our work at the highest tier.
+
+## Stargazers over time
+
+[![Stargazers over time](https://starchart.cc/jart/cosmopolitan.svg)](https://starchart.cc/jart/cosmopolitan)
```
Comment (jart) [inline README.md:282]: I love this service, but it's not reliable and frequently runs out of quota. I would recommend removing this section.

**Review by jart — CHANGES_REQUESTED** [review]

**Comment by jart** [comment]
Actually I just realized that unfortunately the Discord invites expire after a few days. So it's not possible to add it to the README. We need to grow the Discord community to 1000 members before we can go public.

**Comment by i2** [comment]
> 

Discord invite links can last forever. I know as I have it set up for my server:

```
When the server owner or administrator creates an invite link, they can customize the duration for how long the invite is valid.

If no customization is made, the invite expires within 24 hours. However, customization can be made to set the invite to expire after a given period or not to expire at all.
```

Please watch this screen capture I made.

https://user-images.githubusercontent.com/3228126/216786782-12b4fcb3-77e3-44da-a721-77e7ec45023a.mp4

**Pull request closed without merging**